uaclient/cli.py

```diff
@@ -737,6 +737,30 @@ def disable_parser(parser, cfg: config.UAConfig):
     return parser
 
 
+def reboot_required_parser(parser):
+    """Build or extend an arg parser for reboot-required subcomman."""
+    parser.description = "Output the reboot state for the current machine."
+    parser.prog = "reboot-required"
+    # This formatter_class ensures that our formatting below isn't lost
+    parser.formatter_class = argparse.RawDescriptionHelpFormatter
+    parser.description = textwrap.dedent(
+        """\
+        Report the current reboot-required status for the machine.
+
+        This command will output one of the three following states
+        for the machine regarding reboot:
+
+        * no: The machine doesn't require a reboot
+        * yes: The machine requires a reboot
+        * yes-kernel-livepatches-applied: There are only kernel related
+          packages that require a reboot, but Livepatch has already provided
+          patches for the current running kernel. The machine still needs a
+          reboot, but you can assess if the reboot can be performed in the
+          nearest maintenance window.
```

Comment:
I think the cli command should return different codes depending on the state as well

Reply:
My only concern regarding this is that we non-zero return codes might be considered an error, which none of those states represent

Reply:
yeah good point